doc/source/cookbook.rst

```diff
@@ -1226,6 +1226,17 @@ Computation
 Correlation
 ***********
 
+Often it's useful to obtain the lower (or upper) triangular form of a correlation matrix calculated from `DataFrame.corr`.  This can be achieved by passing a boolean mask to `where` as follows:
```

Comment:
I think you can add a `func:` in front of `DataFrame.corr` and double backticks around where.